tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "include": [
+    "./src/utils/assert.js",
+    "./src/utils/assert.test.js",
+    "./src/utils/location.js",
+    "./src/utils/location.test.js",
+    "./src/utils/math.js",
+    "./src/utils/math.test.js",
+    "./src/utils/objects.js",
+    "./src/utils/strings.js",
+    "./src/utils/strings.test.js",
+  ],
+  "compilerOptions": {
+    "incremental": true, /* Save .tsbuildinfo files to allow for incremental compilation */
+    "outDir": "./built",
```

Comment:
Since it's just soft checks, we should disable emitting files from compilation.

This should be replaced with the `noEmit` option.

Reply:
Ah cool.. and i think not using incremental mode as well?

Reply:
Looks like it may be slowing down tsc a bit.  But works for now if that's good with you